ccpp/driver/GFS_diagnostics.F90

```diff
@@ -2296,6 +2296,71 @@ subroutine GFS_externaldiag_populate (ExtDiag, Model, Statein, Stateout, Sfcprop
       enddo
     endif
 
+    if (Model%do_spp) then
+      idx = idx + 1
+      ExtDiag(idx)%axes = 3
+      ExtDiag(idx)%name = 'spp_wts_pbl'
+      ExtDiag(idx)%desc = 'spp pbl perturbation wts'
+      ExtDiag(idx)%unit = 'm/s'
+      ExtDiag(idx)%mod_name = 'gfs_phys'
+      allocate (ExtDiag(idx)%data(nblks))
+      do nb = 1,nblks
+        ExtDiag(idx)%data(nb)%var3 => IntDiag(nb)%spp_wts_pbl(:,:)
+      enddo
```

Comment:
@climbfuji, are these "Intdiag" references OK here?

Reply:
These `IntDiag` references need to be replaced with `Coupling` (lines 2308, 2321, 2334, 2347, 2360). I'll add suggestions that you can accept.

Reply:
Thanks!  Fixed!